### DISCUSSION TITLE
Theia 1.27.0

### DISCUSSION THREAD

**Discussion by mrora** [eclipse-theia/theia#11352] [General] [answered]
Hello, Team!

We recently encountered an issue around disconnecting from and reconnecting to the backend that appears to be addressed by [#11228](https://github.com/eclipse-theia/theia/pull/11228) (i.e. ensuring message channels get disposed and recreated as expected). I see that those changes will be [included in 1.27.0](https://github.com/eclipse-theia/theia/blob/master/CHANGELOG.md#v1270---unreleased).

Theia releases appear to happen around the end of the month. Is that the case for 1.27.0?

Thanks!

**Comment by vince-fugnitto** [comment, marked as answer]
@mrora thank you for the discussion, we generally release on the last Thursday of each month, so `1.27.0` should be released on June 30th.

You should also be able to see the tentative dates when looking at the [milestones](https://github.com/eclipse-theia/theia/milestones).

**Comment by mrora** [comment]
thanks, @vince-fugnitto!
